client/src/search-utils.ts

```diff
@@ -32,3 +32,77 @@ export function useFocusViaKeyboard(
     };
   }, [inputRef]);
 }
+
+const COLLECTION_ITEMS_KEY: string = "collection-items";
```

Comment:
I feel like lines 36 to 108 belong in a separate file, inside the `plus` directory.